src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+---
```

Comment:
Does this doc need a nav entry in `sidenav.json`?

Reply:
In a previous PR, I created a /terraform landing page. I think that page will need a nav entry in the sidenav

Reply:
Perfect!